src/js/models/layerstructure.js

```diff
@@ -586,6 +586,17 @@ define(function (require, exports, module) {
                 !curLayer.isVector &&
                 !curLayer.isBackground &&
                 !curLayer.locked;
+        },
+
+        /**
+         * determine if there are only Shape layers in the current selection
+         *
+         * @return {boolean} if the layer selection contains only vector shapes
+         */
+        "selectedLayersContainOnlyShapeLayers": function () {
+            return this.selected.all(function (layer) {
```

Comment:
Can you use `.every()` here, even if .all works, it's not documented anywhere, and is not conventional. We use `every` everywhere else.